doc/btp_sdp.txt

```diff
@@ -0,0 +1,62 @@
+SDP Service (ID 31)
+=====================
+
+Commands and responses:
+
+        Opcode 0x00 - Error response
+
+        Opcode 0x01 - Read Supported Commands command/response
+
+                Controller Index:       <controller id>
+                Command parameters:     <none>
+                Response parameters:    <supported commands> (variable)
+
+                Each bit in response is a flag indicating if command with
+                opcode matching bit number is supported. Bit set to 1 means
+                that command is supported. Bit 0 is reserved and shall always
+                be set to 0. If specific bit is not present in response (less
+                than required bytes received) it shall be assumed that command
+                is not supported.
+
+                In case of an error, the error response will be returned.
+
+        Opcode 0x02: SDP service search request
+
+                Controller Index:       <controller id>
```

Comment:
while SDP is BR/EDR I'd suggest to have address type here (and in all SDP commands/events)  to (like for all others profiles)

this is mainly to keep BTP consistent,  those commands here will return error if non-bredr type is provided